ml_service/pipelines/diabetes_regression_build_train_pipeline_with_r.py

```diff
@@ -26,15 +26,11 @@ def main():
         print("aml_compute:")
         print(aml_compute)
 
-    run_config = RunConfiguration(conda_dependencies=CondaDependencies.create(
-        conda_packages=['numpy', 'pandas',
-                        'scikit-learn', 'tensorflow', 'keras'],
-        pip_packages=['azure', 'azureml-core',
-                      'azure-storage',
-                      'azure-storage-blob'])
-    )
+    # Create a run configuration environment
+    conda_deps_file = "diabetes_regression/training_dependencies.yml"
+    conda_deps = CondaDependencies(conda_deps_file)
+    run_config = RunConfiguration(conda_dependencies=conda_deps)
     run_config.environment.docker.enabled = True
-    run_config.environment.docker.base_image = "mcr.microsoft.com/mlops/python"
```

Comment:
We need this container with r_essentails

Reply:
we had it essentially to demonstrate the use of the container for training 

Reply:
I've added to the doc instead:

    You will also need to add the
     `r-essentials` Conda packages into `diabetes_regression/scoring_dependencies.yml`
     and `diabetes_regression/training_dependencies.yml`.

I think it's a much more robust solution, and guides R users to the right process for adding the additional packages they will usually need.

Reply:
Tested, training seems to run fine:

```
Starting the daemon thread to refresh tokens in background for process with pid = 137
Entering Run History Context Manager.
[1] "R version 3.6.1 (2019-07-05)"
[1] "Reading file from weight_data.csv"
   height weight
1      79    174
2      63    250
3      75    223
4      75    130
5      70    120
6      76    239
7      63    129
8      64    185
9      59    246
10     80    241
11     79    217
12     65    212
13     74    242
14     71    223
15     61    167
16     78    148
17     75    229
18     75    116
19     75    182
20     72    237
21     72    160
22     79    169
23     67    219
24     61    202
25     65    168
26     79    181
27     81    214
28     78    216
29     59    245
       1        2 
173.6420 222.3347 

Call:
lm(formula = y ~ x)

Coefficients:
(Intercept)            x  
   232.5858      -0.5126  

[1] "Completed"
-rwxrwxrwx 1 root root 1740 Jan 31 20:10 model.rds


The experiment completed successfully. Finalizing run...
Cleaning up all outstanding Run operations, waiting 300.0 seconds
1 items cleaning up...
Cleanup took 0.0007724761962890625 seconds
Starting the daemon thread to refresh tokens in background for process with pid = 137
```

Reply:
agree with you and that's what we showcased in python training pipeline and for R we wanted to demonstrate that one can bring in their base image for training as well :)

Reply:
If we want to showcase it, I think it's better to do that in a doc than buried in a script